parts/linux/cloud-init/artifacts/setup-custom-search-domains.sh

```diff
@@ -6,6 +6,6 @@ source "${CSE_DISTRO_HELPERS_FILEPATH}"
 echo "  dns-search ${CUSTOM_SEARCH_DOMAIN_NAME}" | tee -a /etc/network/interfaces.d/50-cloud-init.cfg
 systemctl_restart 20 5 10 networking
```

Comment:
Technically we should/could also exit from here, but not sure if it's a failure we want to propagate? 

In `cse_main.sh` we do:

```
if [ "${HAS_CUSTOM_SEARCH_DOMAIN}" == "true" ]; then
    "${CUSTOM_SEARCH_DOMAIN_FILEPATH}" > /opt/azure/containers/setup-custom-search-domain.log 2>&1 || exit $ERR_CUSTOM_SEARCH_DOMAINS_FAIL
fi
```

But in the current state of the script, I believe we would only report that error if the last line of the script fails somehow?

Reply:
is this script called during CSE?